mvp-spec/website-design.md

```diff
@@ -0,0 +1,130 @@
+# Web site integration: design
+
+## Read data
+
+This diagram details the steps needed to read existing cookies from Prebid SSO
+- at HTTP level **if the website hosts a middleware** that can trigger HTTP redirects when needed
+- at Javascript level in any case
+  - the Javascript integration relies on the **operator client**, a component responsible for building operator URLs to call,
+  hosted by the website or by a vendor _on the website's behalf_
+  (see [landscape](./landscape.md) for context)
+  - depending on the context, this operator client will call a JSON or "redirect" endpoint on the operator
+
+### Test support of 3rd party cookies
+
+The logic to test if third party cookies are supported and use full page redirect if they aren't is the following:
```

Comment:
Nitpick: I suggest simplifying or rephrasing this sentence by using punctuation.